### PR TITLE
Simplify examples by using latest flatdata version

### PR DESCRIPTION
**Pull request by VeaaC**
Signed-off-by: Christian Vetter <christian.vetter@here.com>

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -15,7 +15,7 @@ byteorder = "1.2"
 bytes = "0.4"
 colored = "1.6"
 failure = "0.1"
-flatdata = { git = "https://github.com/boxdot/flatdata-rs", rev = "df86fda4" }
+flatdata = { git = "https://github.com/heremaps/flatdata", rev = "67aed3f6f91eefa4974ecc89fe394f4830505f37" }
```
Comment (boxdot) [inline Cargo.toml]: I think we should publish, not? Then we can use crates.io here.

**Comment by VeaaC** [comment]
How about it @gferon @boxdot ?

File: Cargo.toml (hunk 1 above)
Comment (boxdot) [reply Cargo.toml]: I updated it to 0.3 myself.

**Review by boxdot — APPROVED** [review]
@VeaaC: If you are ok with my commit on top, feel free to merge.

**Pull request merged**